server/src/main/resources/reference.conf

```diff
@@ -13,7 +13,7 @@ org.scala_lang.index.server {
 app {
    env = "local"
    endpoint = "localhost"
-   port = 8082
+   port = 8080
```

Comment:
what's the reason for modifying it?

Reply:
authentication always redirect to 8080/callback... 
I don't know how the port is passed, I think it's some default one

Reply:
so this setting is not used anywhere?